spatialos/UnrealGDK
# [UNR-2689][MS] Spatial output log will pop up at the end of PIE sessi…

…on. A setting can be changed to disable this.

**Contributions**: We are not currently taking public contributions - see our [contributions](CONTRIBUTING.md) policy. However, we are accepting issues and we do want your [feedback](../README.md#give-us-feedback).

-------

#### Description
Spatial output log will now be displayed on PIE session end. This is similar to message output functionality. There is also a setting to stop the output popping up. If the spatial output is already open, the window will not be modified or pop up again annoyingly.  

#### Release note
Changelog updated

QA Testing:
Open up example project. Ensure that the Spatial output log is not turned on. Start and stop a PIE session. Note that the Spatial output log is now visible. Now close the Spatial output log and disable the "OpenSpatialOutputLogOnPIESessionEnd" inside "Service Settings". Test again. The Spatial output log should not pop up.

#### Primary reviewers
@jessicafalk  @joshuahuburn
